governance/program/tests/program_test/mod.rs

```diff
@@ -185,10 +185,16 @@ impl GovernanceProgramTest {
         let account = Realm {
             account_type: GovernanceAccountType::Realm,
             community_mint: community_token_mint_keypair.pubkey(),
-            council_mint: Some(council_token_mint_keypair.pubkey()),
+
             name,
             reserved: [0; 8],
             authority: Some(realm_authority.pubkey()),
+            config: RealmConfig {
+                council_mint: Some(council_token_mint_keypair.pubkey()),
+                reserved: [0; 8],
+                custodian: Some(realm_authority.pubkey()),
```

Comment:
Once you're actually testing this, it could be better to use a different authority to avoid false positives

Reply:
yeap, done in next PR